scripts/test.ps1

```diff
@@ -197,6 +197,10 @@ function Get-VSTestPath
     $TestPlatformVersion = (([XML](Get-Content $versionsFile)).Project.PropertyGroup.TestPlatformVersion).InnerText
 
     $vsInstallPath = "$PSScriptRoot\..\packages\Microsoft.TestPlatform.$TestPlatformVersion\"
```

Comment:
When is this true? I don't see tp.<version> packages in my packages folder in vstest.

Reply:
If the packages are restored using PackageReference and not NuGet.config, this path would not exists. It's just a temporary resolution since we actually don't use this script in the upcoming arcade switch.

Reply:
I was looking into the wrong project, vstest instead of testfx. In testfx I have those folders and it makes sense now.